### PR TITLE
Pruned repo

### PR DESCRIPTION
**Pull request by clundin55**

### CODE REVIEW

File: database/api/routes.md [hunk 1]
```diff
@@ -1,123 +0,0 @@
-# api/v1/save/
```
Comment (aniehuser) [inline database/api/routes.md]: we should keep this file, usually api's have some sort of documentation

File: scripts/serverSetup.md [hunk 2]
```diff
@@ -1,49 +0,0 @@
-# Getting started with YesWorkflow Web Components
```
Comment (aniehuser) [inline scripts/serverSetup.md:1]: is this information located somewhere else?

Comment (clundin55) [reply scripts/serverSetup.md:1]: Yea most of this is redundant with the README

**Review by aniehuser — CHANGES_REQUESTED** [review]
we shouldnt merge this if we are deleting the API documenation

File: database/api/routes.md (hunk 1 above)
Comment (clundin55) [reply database/api/routes.md]: I added this back

**Pull request merged**